torchvision/transforms/functional_pil.py

```diff
@@ -271,7 +306,15 @@ def affine(img, matrix, interpolation=0, fill=None):
 
 
 @torch.jit.unused
-def rotate(img, angle, interpolation=0, expand=False, center=None, fill=None):
+def rotate(
+    img: Image.Image,
+    angle: float,
+    interpolation: int = 0,
```

Comment:
same here

Reply:
We could change it as you suggested, but again this is inconsistent with functional_tensor.

Reply:
So the reason why I mention this is because on other methods in this file you use the "enum" while in this case you use the integer equivalent (see `perspective` for example). I think the enum will be clearer.

Reply:
Aah cool, I get it. I will rewrite with Enum :smile:

Reply:
yeah sorry I should have been clearer. OK ping me when this is done and I'll approve. Thanks for the effort and sorry for the back and forth. :)